Docs: Org, Team, and User Admin (#30756)
grafana/grafana: docs/sources/manage-users/_index.md

Server Admins can:

- [Manage users]({{< relref "server-admin-manage-users.md" >}})
- [Manage organizations]({{< relref "server-admin-manage-orgs.md" >}})

## Organization

Organizations are groups of users on a server. Users can belong to one or more organizations, but each user must belong to at least one organization.

Data sources, plugins, and dashboards are associated with organizations. This means that you can have a server with two organizations, one with a Prometheus data source and another with an InfluxDB data source. Each organization has separate data and dashboards.

Members of organizations have permissions based on their _role_ in the organization. For more information, refer to [Organization roles]({{< relref "../permissions/organization_roles.md" >}}).

Organization Admins are user accounts that are assigned the Admin role for an organization. They can manage their users and teams in their organization.

Organization Admins can:
- [Manage users]({{< relref "org-admin/_index.md" >}})
- [Manage teams]({{< relref "manage-teams/index.md" >}}) 

## Teams

> Teams are groups of users within the same organization. Teams allow you to grant permissions for a group of users. They are most often used to manage [permissions for folders and dashboards]({{< relref "../permissions/dashboard_folder_permissions.md" >}}). Enterprise users might use them to apply [data source permissions]({{< relref "../enterprise/datasource_permissions.md" >}}).

Reviewer: ```suggestion
Teams are groups of users within the same organization. Teams allow you to grant permissions for a group of users. They are most often used to manage [permissions for folders and dashboards]({{< relref "../permissions/dashboard_folder_permissions.md" >}}). Enterprise users can use them to apply [data source permissions]({{< relref "../enterprise/datasource_permissions.md" >}}).
```

> Teams are mostly managed by Organization Admins, but if the Grafana server setting [editors_can_admin]({{< relref "../administration/configuration.md#editors_can_admin" >}}) is applied, then users assigned the Team Admin role can also manage teams in their organization and users assigned to their teams.

Reviewer 2: ```suggestion
Teams are mostly managed by Organization Admins. However, if the Grafana server setting [editors_can_admin]({{< relref "../administration/configuration.md#editors_can_admin" >}}) is applied, then the users assigned the Team Admin role can also manage teams in their organization, and users assigned to their teams.
```

Team Admins can [Manage teams]({{< relref "manage-teams/index.md" >}}).

## Users

> Users are named accounts in Grafana with granted permissions to access resources throughout Grafana. All users can manage their own user accounts to a certain extent.

Reviewer 2: ```suggestion
Users are named accounts in Grafana with granted permissions to access resources throughout Grafana. All users can manage their own accounts to a limited extent.
```

Teams are groups of users within the same organization. Teams allow you to grant permissions for a group of users. They are most often used to manage [permissions for folders and dashboards]({{< relref "../permissions/dashboard_folder_permissions.md" >}}). Enterprise users might use them to apply [data source permissions]({{< relref "../enterprise/datasource_permissions.md" >}}).

> Teams are mostly managed by Organization Admins. However, if the Grafana server setting [editors_can_admin]({{< relref "../administration/configuration.md#editors_can_admin" >}}) is applied, then the users assigned the Team Admin role can also manage teams in their organization, and users assigned to their teams.

Reviewer: ```suggestion
Teams are mostly managed by Organization Admins. However, if the Grafana server setting [editors_can_admin]({{< relref "../administration/configuration.md#editors_can_admin" >}}) is applied, users who are assigned the Team Admin role can also manage teams in their organization as well as users assigned to their teams.
```
